tools/cellxgene_census_builder/tests/test_release_cleanup.py

```diff
@@ -34,6 +36,18 @@ def tag_days_old(days_old: int) -> str:
         "soma": {"uri": f"{S3_PREFIX}{TAG_100D_OLD}/soma/", "s3_region": "us-west-2"},
         "h5ads": {"uri": f"{S3_PREFIX}{TAG_100D_OLD}/h5ads/", "s3_region": "us-west-2"},
     },
+    "stable": TAG_LTS_A,
+    TAG_LTS_A: {  # mock LTS release with an alias, but without a do_not_delete flag
+        "release_build": TAG_LTS_A,
+        "soma": {"uri": f"{S3_PREFIX}{TAG_LTS_A}/soma/", "s3_region": "us-west-2"},
+        "h5ads": {"uri": f"{S3_PREFIX}{TAG_LTS_A}/h5ads/", "s3_region": "us-west-2"},
+    },
+    TAG_LTS_B: {  # mock LTS release lacking an alias, but with a do_not_delete flag
+        "release_build": TAG_LTS_B,
+        "soma": {"uri": f"{S3_PREFIX}{TAG_LTS_B}/soma/", "s3_region": "us-west-2"},
+        "h5ads": {"uri": f"{S3_PREFIX}{TAG_LTS_B}/h5ads/", "s3_region": "us-west-2"},
+        "do_not_delete": True,
+    },
```

Comment:
Is there a test verifying that `TAG_LTS_B` is not deleted? Hard to tell from the just the PR diff.  Maybe add an explicit `assert` somewhere showing `TAG_LTS_B` is not deleted?